packages/vitest/src/node/plugins/index.ts

```diff
@@ -65,9 +70,12 @@ export async function VitestPlugin(options: UserConfig = {}, ctx = new Vitest())
 
         (options as ResolvedConfig).defines = defines
 
-        const open = preOptions.ui && preOptions.open
-          ? preOptions.uiBase ?? '/__vitest__/'
-          : undefined
+        let open: string | boolean | undefined
+
+        if (preOptions.ui && preOptions.open)
+          open = preOptions.uiBase ?? '/__vitest__/'
+        else if (preOptions.web)
```

Comment:
should be `preOptions.browser`

I'm checking it rn, open my own branch or just work directly here?

Reply:
Nice catch, you can directly commit to this branch